Agora-LinuxServer-Recording/README.md

```diff
@@ -568,8 +570,8 @@ Once the signal stops, leave the channel using `recorder.leaveChannel()` and rel
 
 ## Resources
 - See full API documentation in the [Document Center](https://docs.agora.io/en/)
-- [File bugs about this sample](https://github.com/AgoraIO/Basic-Recording/issues)
+- [File bugs about this sample](https://dashboard.agora.io)

```

Comment:
why? i believe the old link is correct

Reply:
@plutoless I have fixed the erroneous link and updated this branch.